stress/shell/src/main/java/alluxio/stress/cli/UfsIOBench.java

```diff
@@ -43,6 +45,7 @@
 public class UfsIOBench extends Benchmark<IOTaskResult> {
   private static final Logger LOG = LoggerFactory.getLogger(UfsIOBench.class);
   private static final int BUFFER_SIZE = 1024 * 1024;
+  private static final String TEST_DIR_NAME = "iotest";
```

Comment:
Maybe the `TEST_DIR_NAME` should be a special value ? `"iotest"` will be already exists.